theano/compile/tests/test_profiling.py

```diff
@@ -101,7 +100,8 @@ def test_ifelse(self):
             big_mat1 = 10
             big_mat2 = 11
 
-            out = f_ifelse(val1, val2, big_mat1, big_mat2)
```

Comment:
You are removing the test by commenting this out. You should rather call it without assigning it to `out` so that flake8 doesn't complain.